### DISCUSSION TITLE
Undo in Selection

### DISCUSSION THREAD

**Discussion by trev-dev** [meow-edit/meow#293] [Q&A]
I ended up rebinding my `meow-undo-in-selection` keybind to `undo-redo`. I don't really understand the undo-in-selection feature and how to use it. Can anyone shed light on that?

**Comment by leira** [comment]
I just found meow, so I'm just guessing. Is `meow-undo-in-selection` to undo the current selection operation, and to go back the previous selections? If so, then this is a missing feature I really wanted when I use Kakoune.

**Reply by trev-dev** [reply]
The `meow-pop-selection` command does what you're describing :)
